internal/controller/lease_controller_test.go

```diff
@@ -274,6 +305,21 @@ var testExporter3DutB = &jumpstarterdevv1alpha1.Exporter{
 	},
 }
 
+func setExporterCondition(ctx context.Context, name string, status metav1.ConditionStatus) {
```

Comment:
```suggestion
func setExporterReadyConditions(ctx context.Context, name string, status metav1.ConditionStatus) {
```

Reply:
Renamed to `setExporterOnlineConditions` following the condition rename.